src/bootstrap/src/core/config/config.rs

```diff
@@ -2777,13 +2777,9 @@ impl Config {
         }
 
         // Warn if there were changes to the compiler or standard library since the ancestor commit.
-        let has_changes = !t!(helpers::git(Some(&self.src))
-            .args(["diff-index", "--quiet", &commit])
-            .arg("--")
-            .args([self.src.join("compiler"), self.src.join("library")])
-            .as_command_mut()
-            .status())
-        .success();
+        let dirs = vec![self.src.join("compiler"), self.src.join("library")];
```

Comment:
This might be insufficient with respect to beta bumps, e.g. https://github.com/rust-lang/rust/pull/131560#issuecomment-2408863576. Maybe this also need `src/version`? I'm not sure.

Reply:
I'm trying to figure out what bump to 1.84 meaning, and what 'beta bumps' refer to? (These may be some pre-knowledge).

Reply:
https://forge.rust-lang.org/release/process.html#bump-the-stable-version-number-t-6-days-friday-the-week-before